hoot-core/src/main/cpp/hoot/core/ops/RemoveRoundabouts.cpp

```diff
@@ -74,23 +74,32 @@ void RemoveRoundabouts::removeRoundabouts(std::vector<RoundaboutPtr> &removed)
 
   // Mangle (in a good way) ways that may cross our roundabouts, provided there
   // is no 'sibling' roundabout in the secondary dataset
-  for (size_t i = 0; i < removed.size(); i++)
+  QVector<bool> foundSibling(removed.size(), false);
+  for (size_t i = 0; i < removed.size() - 1; i++)
```

Comment:
This is really where the big change was, don't have the last roundabout compare itself to itself and then not "mangle".  Doesn't make sense.  But if it doesn't already have a sibling it is "mangled" at the bottom of the new code.